code/lib/api/src/typings.d.ts

```diff
@@ -1,2 +1,3 @@
 declare module 'global';
 declare module 'preval.macro';
+declare module '@storybook/semver';
```

Comment:
Why did storybook fork semver?  It looks like it's 36 commits behind the upstream package, but no commits ahead.  https://github.com/storybookjs/browser-semver.  If we could use the normal semver package, I think we could also use `@types/semver` instead of having to do the above.  Though, I'd say that's probably a separate PR.

Reply:
See https://github.com/storybookjs/storybook/issues/19197

Reply:
You're a step ahead of me.  :-D

Reply:
Great minds think alike!